stacks-signer/src/v0/signer.rs

```diff
@@ -344,6 +347,23 @@ impl Signer {
             return;
         }
 
+        if block_proposal
+            .block
+            .header
+            .timestamp
+            .saturating_add(self.block_proposal_max_age_secs)
+            < get_epoch_time_secs()
+        {
+            // Block is too old. Drop it with a warning. Don't even bother broadcasting to the node.
+            warn!("{self}: Received a block proposal that is more than {} secs old. Ignoring...", self.block_proposal_max_age_secs;
+                "block_id" => %block_proposal.block.block_id(),
```

Comment:
Could you include the `signer_sighash` in here? I often am looking through the signer logs for that hash.

Reply:
sure thing!